docs/demo.md

```diff
@@ -1,73 +1,325 @@
 ---
 id: demo
-title: components demo
+title: Components demo
```

Comment:
https://libra-website.oriooctopus.vercel.app/docs/demo

You can see the demo here. There are probably a couple of tweeks that have to be done, so let me know about any behavior that you feel is undesirable. I'll also show this to scott